### ISSUE TITLE
executables not found in custom image

### ISSUE THREAD

**Issue by moretrim** [JJ/alpine-perl6#24]
## Starting off with curl

Running off a custom image based on `alpine-raku:latest` as of this writing, it appears `curl` is not found. I started with something along the following in my CI logs:

```interactive
sh: curl: not found
```

(Using `sh` as entrypoint.)

In my case I can tweak my custom image to grab curl:

```dockerfile
FROM jjmerelo/alpine-raku:latest

USER root
RUN ["apk", "add", "--no-cache", "curl"]
```

Already the README mentions that curl is included, so I thought something might be up. 

## But there’s more

Unfortunately at this point it turned out that zef is not found either. I can also add it to be installed, but this is starting to feel wrong. In fact if I do this, installing a Raku package through zef results in yet another missing executable (that was supposed to have been provided by the package).

I had a look at `$PATH` but I’m not sure what I’m supposed to find there or not:

```session
/home/raku/.raku/bin:/home/raku/.raku/share/perl6/site/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

**Comment by JJ** [comment]
Right. It was eliminated in the latest version... If it's positively needed, I can put it back there, but I'd rather eliminate it from the documentation. Would that be OK for you?
WRT zef, the problem is that you need to use the `raku` user to find it, since it's installed in it. Raku binaries are accessible from root, but zef uses a library that's only included with the user.

**Comment by moretrim** [comment]
> If it's positively needed, I can put it back there, but I'd rather eliminate it from the documentation. Would that be OK for you?

It’s fine with me if it’s just removed from the documentation. Adding the things I need is why I have the custom image to begin with.

> […] Raku binaries are accessible from root, but zef uses a library that's only included with the user.

Ah, I must have set up my entry point poorly. 👍 This addresses all my concern, thanks.

**Comment by JJ** [comment]
It's a more subtle problem... The problem is where's `$HOME`. Check out [this question in SO](https://stackoverflow.com/questions/70410854/where-does-the-repo-dynamic-variable-obtain-its-values-and-how-to-change-alte) I just made. I'm preparing anyway a set of containers specifically set up for GitHub actions as I'm writing this... 
Meanwhile a workaround is simply to set `$HOME` to where the installation actually is, `/home/raku`. I've done it [here](https://github.com/JJ/p6-pod-load/blob/4b37106c89139a2fcc6d75e02865f9af3ecf7c59/.github/workflows/test.yaml). Let me know if that works for you for the time being.

**Comment by moretrim** [comment]
> Let me know if that works for you for the time being.

Casual investigation on my part suggests I may also have unrelated things to fix before I can give the go-ahead.. With the end of the year in sight I will not have time to investigate this fully. I would suggest you close this issue at your own convenience.
